views/pages/attachTags.ejs

```diff
@@ -0,0 +1,213 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
+    <meta name="description" content="">
+    <meta name="author" content="">
+
+    <% include ../partials/head.ejs %>
+
+    <script type="text/javascript" src="/utils.js" charset="utf-8"></script>
+</head>
+
+<body>
+
+<% include ../partials/menu.ejs %>
+
+<div class="container bs-docs-container">
+    <div class="col-md-12">
+        <ol class="breadcrumb" style="margin-top: 10px;">
+            <li><a href="/">Home</a></li>
+            <li><a href="/project/<%= project.id %>"><%= project.name %></a></li>
+            <li><a href="/project/<%= project.id %>/tags">Tags</a></li>
+            <li class="active">Attach</li>
+        </ol>
+    </div>
+
+    <div class="col-md-12">
+        <div class="alert alert-info">
+            Select tags that you would like to attach to <strong><%= project.name %></strong>.
+            <p>You can also create new tags or edit/delete the existing ones. Please note that those changes will affect tags globally.</p>
+        </div>
+    </div>
+
+    <div class="row">
+        <div class="col-md-10 col-md-offset-1">
+            <div id="custom-toolbar">
+                <div class="form-inline" role="form">
+                    <button id="submitBtn" href="#" class="btn btn-success">
+                        <i class="glyphicon glyphicon-ok"></i>
+                        Attach selected
```

Comment:
@xjedam No. I guess we could put tables from `attachTags.ejs` and `editTags.ejs` into a single screen. I haven't tried it. Is that what you have in mind?